sdk/keyvault/keyvault-keys/src/index.ts

```diff
@@ -847,7 +863,7 @@ export class KeyClient {
       );
       continuationState.continuationToken = currentSetResponse.nextLink;
       if (currentSetResponse.value) {
-        yield currentSetResponse.value.map(this.getKeyPropertiesFromKeyItem);
+        yield currentSetResponse.value.map(this.getKeyPropertiesFromKeyItem, this);
```

Comment:
I had forgotten that map requires you to specify this, if the method passed uses this. Huh

Reply:
The recent changes due to the feedback I've received have made this change unnecessary. However, I believe leaving this is still valuable since it will save debugging time in the future, if we eventually call members of this from getKeyPropertiesFromKeyItem, and the other functions we're passing to map methods.